docs/getting-started.md

```diff
@@ -36,6 +36,19 @@ A solution to the problem is to pass the provider ID (aka droplet ID) via the ku
 
 DigitalOcean's managed Kubernetes offering DOKS sets the provider ID on each worker node kubelet instance.
 
+### Managed firewall handling for public access
+
+`digitalocean-cloud-controller-manager` can manage a dedicated [DigitalOcean Cloud Firewall](https://www.digitalocean.com/docs/networking/firewalls/) to dynamically allow access to NodePorts. A controller watches over Services and modifies the inbound rules of a firewall to permit access to the target NodePorts, and likewise close down access again if a Service is deleted or its type changed to something other than `NodePort`. (Note that DigitalOcean Load-Balancers access the cluster over the VPC interface and as such are not managed by this particular firewall for now.)
+
+By default, no firewall will be managed. To enable firewall management, the following environment variables need to be defined:
+
+* `PUBLIC_ACCESS_FIREWALL_NAME`: the name of the firewall to use.
+* `PUBLIC_ACCESS_FIREWALL_TAGS`: a comma-separated list of tags that match the worker droplets the firewall should target.
+
+Managed firewalls should **not** be modified directly as such changes will be reverted eventually, including recreation of the firewall should it ever be found missing.
```

Comment:
Everything looks great @timoreimann!

One super small grammar nit. I believe that the word `recreation` should be `re-creation` when referring to the act of creating something again.

Reply:
Makes sense, done.